### ISSUE TITLE
Comments inside array definitions cause a crash

### ISSUE THREAD

**Issue by cmancone** [prettier/plugin-php#258]
Input:
```
<?php
$array = [
    'key1'      => 'value1',
    // a comment inside an array definition
    'key2'      => 'value2',
]
```
```
yarn run v1.5.1
$ prettier --plugin=. --parser=php test.php

[error] test.php: Error: Comment "// a comment inside an array definition" was not printed. Please report this error!
[error]     at astComments.forEach.comment (/var/www/prettier/plugin-php/node_modules/prettier/index.js:54:13)
[error]     at Array.forEach (<anonymous>)
[error]     at ensureAllCommentsPrinted (/var/www/prettier/plugin-php/node_modules/prettier/index.js:52:15)
[error]     at formatWithCursor (/var/www/prettier/plugin-php/node_modules/prettier/index.js:117:3)
[error]     at Object.formatWithCursor (/var/www/prettier/plugin-php/node_modules/prettier/index.js:390:12)
[error]     at format (/var/www/prettier/plugin-php/node_modules/prettier/src/cli/util.js:157:19)
[error]     at eachFilename (/var/www/prettier/plugin-php/node_modules/prettier/src/cli/util.js:382:16)
[error]     at filePaths.forEach.filePath (/var/www/prettier/plugin-php/node_modules/prettier/src/cli/util.js:322:7)
[error]     at Array.forEach (<anonymous>)
[error]     at eachFilename (/var/www/prettier/plugin-php/node_modules/prettier/src/cli/util.js:321:15)
error An unexpected error occurred: "Command failed.
Exit code: 2
Command: sh
Arguments: -c prettier --plugin=. --parser=php test.php
```

**Comment by mgrip** [comment]
thanks @cmancone - think this should be fixed now that #255 is merged

**Comment by mgrip** [comment]
confirmed - closing this out
